aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/diamond_clicker.move

```diff
@@ -1,7 +1,40 @@
 //# init --addresses Alice=0xf75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6
 //#      --private-keys Alice=56a26140eb233750cd14fb168c3eb4bd0782b099cde626ec8aff7f3cceb6364f
 
-//# publish
+//# print-bytecode --input=module
```

Comment:
Why do we want to print bytecode here? I ask because these kind of outputs in baseline are typically a pain as they are outside the third-party tree and are very fickle (minor changes to the compiler can change bytecode emitted).

Reply:
I kind of wanted to see if print-bytecode worked.  Also, there's a bug that needs investigating, though it happens in both V1 and V2.  The call to debug::print  is causing a typecheck error.  I'll file a bug to follow up on that.
See #13191.